src/DefaultImageOptimizer.php

```diff
@@ -17,6 +17,8 @@ public function optimizeImage(string $pathToImage, string $pathToOutput = null)
         $optimizerChain = OptimizerChainFactory::create();
         $optimizerChain->optimize($pathToImage, $pathToOutput);
         $this->convert($pathToOutput, $pathToOutput);
+
+        return true;
```

Comment:
No tendría sentido hacer `return $this->convert(...);`

y en convert hacer: 

```php
$result = shell_exec($command); // devuelve string o null si el comando falla. 

return $result !== null;
```

Reply:
Listo!